OrbitQt/orbitmainwindow.ui

```diff
@@ -408,11 +407,6 @@
     <string>Show Includes Util</string>
    </property>
   </action>
-  <action name="actionDiff">
-   <property name="text">
-    <string>Diff</string>
-   </property>
-  </action>
```

Comment:
not hiding this behind flag?

Reply:
No, I'm removing it altogether based on [b/156071939#comment2](http://b/156071939#comment2) (which I agree with).